src/main/com/fulcrologic/rad/rendering/semantic_ui/report.cljc

```diff
@@ -22,10 +22,11 @@
     (when (seq row-actions)
       (div :.ui.buttons
         (map-indexed
-          (fn [idx {:keys [label reload? visible? disabled? action]}]
+          (fn [idx {:keys [label reload? visible? disabled? action classes] :as props}]
             (when (or (nil? visible?) (?! visible? report-instance row-props))
               (dom/button :.ui.button
                 {:key      idx
+                 :classes  (into [] classes)
```

Comment:
Generally a good idea to support lambdas on these using `?!` as on line 26.  Also, the data type and name for this should probably be standardized. I think I've already got something like this for other things, and moving to a vector like this is possibly ok, but it's going to lead to API bloat. 

If you do a git grep on `class` on the source you'll see places where I've defined things like `::controls-class` and such.  I guess I'm not opposed, but perhaps we should standardize on `css-classes` instead, since `class` is an overloaded word.  Bummer I wasn't a bit more careful with this tidbit. Prob. should have left this one alpha a bit longer.